src/main/java/mpern/sap/commerce/build/util/Extension.java

```diff
@@ -1,20 +1,43 @@
 package mpern.sap.commerce.build.util;
 
 import java.nio.file.Path;
+import java.util.List;
 import java.util.Objects;
 import java.util.StringJoiner;
 
 public class Extension {
+
     public final String name;
     public final Path directory;
 
+    /** Relative location of the extension compared to the hybris/bin folder */
+    public String relativeLocation;
```

Comment:
would be nice if this is also a `Path` 

not really a fan of "stringly typed" code

Reply:
See comment, I avoided it as I am not sure of Windows side effects.